plugins/org-react/src/components/GroupListPicker/GroupListPicker.tsx

```diff
@@ -108,6 +110,8 @@ export const GroupListPicker = (props: GroupListPickerProps) => {
           renderInput={params => (
             <TextField
               {...params}
+              // eslint-disable-next-line jsx-a11y/no-autofocus
+              autoFocus
```

Comment:
This is only rendered when the TextField is opened right? Thinking about cases where we might have things like search in the same page.

Reply:
yes, exactly. The a11y concern for this is that the user does not understand the context switch, but in this case it's an active action to actually clicking or pressing the GroupListPicker button. Inputting or scrolling down is the next logical thing to do